doc/developer_guide/psyir.rst

```diff
@@ -300,10 +365,9 @@ for CodeBlocks. However, the purpose of the PSyIR is to capture code
 concepts that are relevant for performance, not all aspects of a code,
 therefore it is likely that that CodeBlocks will continue to be an
```

Comment:
Not your code but we have "that that". Could you please fix this.

Reply:
Fixed

Reply:
Thanks